src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -1188,3 +1190,17 @@ def parse_otel_span_status_code(value: int) -> str:
         return "Ok"
     else:
         return "Error"
+
+
+def extract_workspace_triad_from_trace_provider(trace_provider: str) -> AzureMLWorkspaceTriad:
+    match = re.match(AZURE_WORKSPACE_REGEX_FORMAT, trace_provider)
+    if not match or len(match.groups()) != 5:
+        raise ValueError(
+            "Malformed trace provider string, expected azureml:/subscriptions/<subscription_id>/"
```

Comment:
azureml://subscripts.... (also please help refine the connection one)

Reply:
updated here and there (connection)